go-controller/pkg/node/node.go

```diff
@@ -138,9 +138,16 @@ func isOVNControllerReady(name string) (bool, error) {
 	err = wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
 		stdout, _, err := util.RunOVSOfctl("dump-aggregate", "br-int")
 		if err != nil {
+			klog.Errorf("Error dumping aggregate flows: %v "+
+				"for node: %s", err, name)
 			return false, nil
 		}
-		return !strings.Contains(stdout, "flow_count=0"), nil
+		ret := strings.Contains(stdout, "flow_count=0")
+		if !ret {
```

Comment:
Clarification: Looking at the message logged in L147, I am wondering if this should be `ret` instead of `!ret` at L146?

Reply:
Yes. Good catch. Will fix it.